### PR TITLE
New and updated corner rounding tokens

### PR DESCRIPTION
**Pull request by nabuhasan**
## Description

These changes consist of updating the existing corner-radius tokens, adding net-new global corner radius tokens, and introducing new alias names that point to these global values. Desktop and mobile scales share the same values. 

**Updated** values for existing corner rounding tokens:

- corner-radius-75
- corner-radius-100
- corner-radius-200

**New** corner rounding tokens and aliases:

- corner-radius-0
- corner-radius-300
- corner-radius-400
- corner-radius-500
- corner-radius-600
- corner-radius-700
- corner-radius-800
- corner-radius-1000 (number token) 
- corner-radius-none
- corner-radius-small-default 
- corner-radius-medium-default 
- corner-radius-large-default 
- corner-radius-extra-large-default
- corner-radius-full
- corner-radius-small-size-small
- corner-radius-small-size-medium
- corner-radius-small-size-large
- corner-radius-small-size-extra-large
- corner-radius-medium-size-extra-small
- corner-radius-medium-size-small
- corner-radius-medium-size-medium
- corner-radius-medium-size-large
- corner-radius-medium-size-extra-large

## Motivation and context

More on Spectrum 2 rounding: [https://s2.spectrum.corp.adobe.com/page/object-styles/#rounding](url)

From Matt K (designer who worked on these values): In Spectrum 2, rounding also applies to the different t-shirt sized components and varies based on a Major Second logarithmic scale, rounded to whole numbers to avoid using half-pixels. This allows for components to retain a consistent and identifiable shape at all sizes. Because of this there are now component size specific tokens like _corner-radius-small-size-medium_ or _corner-radius-medium-size-extra-large_ to help us keep track of the nuance within tshirt sizes.

## Types of changes

- [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue)
- [X] Minor (add a new token, changing a value; non-breaking change which adds functionality)
- [ ] Major (deleting a token, changing token value type; fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.)

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7617085554)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-1000
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-1000
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-0
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-0
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-300
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-300
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-400
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-400
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-500
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-500
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-600
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-600
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-700
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-700
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-800
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-800
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-none
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-none
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-extra-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-extra-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-full
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-full
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-extra-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-extra-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-extra-large
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (46) corner-radius-1000/desktop, corner-radius-1000/mobile, corner-radius-0/desktop, corner-radius-0/mobile, corner-radius-300/desktop, corner-radius-300/mobile, corner-radius-400/desktop, corner-radius-400/mobile, corner-radius-500/desktop, corner-radius-500/mobile, corner-radius-600/desktop, corner-radius-600/mobile, corner-radius-700/desktop, corner-radius-700/mobile, corner-radius-800/desktop, corner-radius-800/mobile, corner-radius-none/desktop, corner-radius-none/mobile, corner-radius-small-default/desktop, corner-radius-small-default/mobile, corner-radius-medium-default/desktop, corner-radius-medium-default/mobile, corner-radius-large-default/desktop, corner-radius-large-default/mobile, corner-radius-extra-large-default/desktop, corner-radius-extra-large-default/mobile, corner-radius-full/desktop, corner-radius-full/mobile, corner-radius-small-size-small/desktop, corner-radius-small-size-small/mobile, corner-radius-small-size-medium/desktop, corner-radius-small-size-medium/mobile, corner-radius-small-size-large/desktop, corner-radius-small-size-large/mobile, corner-radius-small-size-extra-large/desktop, corner-radius-small-size-extra-large/mobile, corner-radius-medium-size-extra-small/desktop, corner-radius-medium-size-extra-small/mobile, corner-radius-medium-size-small/desktop, corner-radius-medium-size-small/mobile, corner-radius-medium-size-medium/desktop, corner-radius-medium-size-medium/mobile, corner-radius-medium-size-large/desktop, corner-radius-medium-size-large/mobile, corner-radius-medium-size-extra-large/desktop, corner-radius-medium-size-extra-large/mobile
CHANGED VALUES: (5) corner-radius-75/desktop, corner-radius-200/desktop, corner-radius-75/mobile, corner-radius-100/mobile, corner-radius-200/mobile impacting 4 tokens
```
</p></details>

TEST COMPLETE

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json [hunk 1]
```diff
@@ -747,6 +769,237 @@
...
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-small-size-small",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-small-size-medium": {
+    "value": "{corner-radius-small-default}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json:916]: @nabuhasan this value should be corner-radius-100 (they should all point to the global tokens, not alias tokens)

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json [hunk 2]
```diff
@@ -747,6 +769,237 @@
...
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-medium-size-small",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-medium-size-medium": {
+    "value": "{corner-radius-medium-default}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json:971]: @nabuhasan this should be `corner-radius-500`

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json [hunk 3]
```diff
@@ -747,6 +769,237 @@
...
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-medium-size-large",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-medium-size-extra-large": {
+    "value": "{corner-radius-large-default}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json:993]: @nabuhasan this should be `corner-radius-700`

Comment (lynnhao) [reply src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json:993]: oops I accidentally started a review here but not sure what that means 😅

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json [hunk 4]
```diff
@@ -747,6 +769,237 @@
...
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-small-size-small",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-small-size-medium": {
+    "value": "{corner-radius-small-default}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json:916]: this should be `corner-radius-100`

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json [hunk 5]
```diff
@@ -747,6 +769,237 @@
...
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-medium-size-small",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-medium-size-medium": {
+    "value": "{corner-radius-medium-default}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json:971]: this should be `corner-radius-500`

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json [hunk 6]
```diff
@@ -747,6 +769,237 @@
...
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-medium-size-large",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-medium-size-extra-large": {
+    "value": "{corner-radius-large-default}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json:993]: this should be `corner-radius-700`

**Review by lynnhao — CHANGES_REQUESTED** [review]
I compared these to the values and noticed a few that need to be remapped to global tokens. Other than that they all look great!

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7630814833)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-1000
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-1000
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-0
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-0
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-300
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-300
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-400
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-400
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-500
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-500
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-600
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-600
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-700
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-700
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-800
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-800
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-none
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-none
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-extra-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-extra-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-full
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-full
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-extra-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-extra-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-extra-large
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (46) corner-radius-1000/desktop, corner-radius-1000/mobile, corner-radius-0/desktop, corner-radius-0/mobile, corner-radius-300/desktop, corner-radius-300/mobile, corner-radius-400/desktop, corner-radius-400/mobile, corner-radius-500/desktop, corner-radius-500/mobile, corner-radius-600/desktop, corner-radius-600/mobile, corner-radius-700/desktop, corner-radius-700/mobile, corner-radius-800/desktop, corner-radius-800/mobile, corner-radius-none/desktop, corner-radius-none/mobile, corner-radius-small-default/desktop, corner-radius-small-default/mobile, corner-radius-medium-default/desktop, corner-radius-medium-default/mobile, corner-radius-large-default/desktop, corner-radius-large-default/mobile, corner-radius-extra-large-default/desktop, corner-radius-extra-large-default/mobile, corner-radius-full/desktop, corner-radius-full/mobile, corner-radius-small-size-small/desktop, corner-radius-small-size-small/mobile, corner-radius-small-size-medium/desktop, corner-radius-small-size-medium/mobile, corner-radius-small-size-large/desktop, corner-radius-small-size-large/mobile, corner-radius-small-size-extra-large/desktop, corner-radius-small-size-extra-large/mobile, corner-radius-medium-size-extra-small/desktop, corner-radius-medium-size-extra-small/mobile, corner-radius-medium-size-small/desktop, corner-radius-medium-size-small/mobile, corner-radius-medium-size-medium/desktop, corner-radius-medium-size-medium/mobile, corner-radius-medium-size-large/desktop, corner-radius-medium-size-large/mobile, corner-radius-medium-size-extra-large/desktop, corner-radius-medium-size-extra-large/mobile
CHANGED VALUES: (5) corner-radius-75/desktop, corner-radius-200/desktop, corner-radius-75/mobile, corner-radius-100/mobile, corner-radius-200/mobile impacting 4 tokens
```
</p></details>

TEST COMPLETE

**Review by lynnhao — APPROVED** [review]
Double-checked the values, looks great! 💯

**Review by GarthDB — APPROVED** [review]

**Comment by mrcjhicks** [comment]
**[SYNCING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7647674568)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-1000
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-1000
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-0
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-0
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-300
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-300
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-400
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-400
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-500
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-500
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-600
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-600
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-700
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-700
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-800
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-800
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-none
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-none
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-extra-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-extra-large-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-full
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-full
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-small-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-small-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-extra-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-extra-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-small
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/desktop.corner-radius-medium-size-extra-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout/mobile.corner-radius-medium-size-extra-large
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (46) corner-radius-1000/desktop, corner-radius-1000/mobile, corner-radius-0/desktop, corner-radius-0/mobile, corner-radius-300/desktop, corner-radius-300/mobile, corner-radius-400/desktop, corner-radius-400/mobile, corner-radius-500/desktop, corner-radius-500/mobile, corner-radius-600/desktop, corner-radius-600/mobile, corner-radius-700/desktop, corner-radius-700/mobile, corner-radius-800/desktop, corner-radius-800/mobile, corner-radius-none/desktop, corner-radius-none/mobile, corner-radius-small-default/desktop, corner-radius-small-default/mobile, corner-radius-medium-default/desktop, corner-radius-medium-default/mobile, corner-radius-large-default/desktop, corner-radius-large-default/mobile, corner-radius-extra-large-default/desktop, corner-radius-extra-large-default/mobile, corner-radius-full/desktop, corner-radius-full/mobile, corner-radius-small-size-small/desktop, corner-radius-small-size-small/mobile, corner-radius-small-size-medium/desktop, corner-radius-small-size-medium/mobile, corner-radius-small-size-large/desktop, corner-radius-small-size-large/mobile, corner-radius-small-size-extra-large/desktop, corner-radius-small-size-extra-large/mobile, corner-radius-medium-size-extra-small/desktop, corner-radius-medium-size-extra-small/mobile, corner-radius-medium-size-small/desktop, corner-radius-medium-size-small/mobile, corner-radius-medium-size-medium/desktop, corner-radius-medium-size-medium/mobile, corner-radius-medium-size-large/desktop, corner-radius-medium-size-large/mobile, corner-radius-medium-size-extra-large/desktop, corner-radius-medium-size-extra-large/mobile
CHANGED VALUES: (5) corner-radius-75/desktop, corner-radius-200/desktop, corner-radius-75/mobile, corner-radius-100/mobile, corner-radius-200/mobile impacting 4 tokens
```
</p></details>

WROTE: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json`
WROTE: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json`
WROTE: `/spectrum-tokens/packages/tokens/src/layout.json`

SYNC COMPLETE

[Spectrum Tokens Pull Request](https://github.com/adobe/spectrum-tokens/pull/247)
[Tokens Studio Pull Request](https://github.com/adobe/spectrum-tokens-studio-data/pull/42)

**Pull request merged**